style.css

```diff
@@ -1,98 +1,280 @@
 :root {
-  --branding-color: #ff6b00;
-  --primary-color: #fff5e1;
-  --secondary-color: #fff;
-  --heading-font-family: "Crete Round", serif;
+  --default-background-color: #1c1a19;
+  --default-font-color: #ff6b00;
+  --primary-font-color: #fff5e1;
+  --secondary-font-color: #fff;
   --default-font-family: "Poppins", sans-serif;
+  --primary-font-family: "Crete Round", serif;
   --secondary-font-family: "Roboto", sans-serif;
-}
-
-body {
-  background: #1c1a19;
+  --third-font-family: "Inter", sans-serif;
 }
 
 @media (min-width: 375px) {
-  h1 {
-    color: var(--branding-color);
-    font-family: var(--heading-font-family);
+  p {
+    color: #091e42;
+    font-family: var(--third-font-family);
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 24px;
+  }
+
+  .about-me {
+    background: var(--default-background-color);
+    padding: 120px 24px;
+  }
+
+  .about-me h1 {
+    color: var(--default-font-color);
+    font-family: var(--primary-font-family);
     font-size: 56px;
+    font-style: normal;
     font-weight: 400;
     line-height: 64px;
     letter-spacing: 0.37px;
   }
 
-  h3 {
-    color: var(--primary-color);
+  .about-me h3 {
+    color: var(--primary-font-color);
+    font-family: var(--secondary-font-family);
     font-size: 24px;
+    font-style: normal;
     font-weight: 700;
     line-height: 32px;
-    font-family: var(--secondary-font-family);
   }
 
-  p {
-    color: var(--secondary-color);
+  .about-me p {
+    color: var(--secondary-font-color);
     font-family: var(--default-font-family);
     font-size: 16px;
+    font-style: normal;
     font-weight: 400;
     line-height: 24px;
-    margin-top: 8px;
   }
 
-  nav {
-    height: 48px;
-    padding: 2px 24px 2px 12px;
+  .social-links {
+    padding: 8px;
   }
 
-  nav ul {
-    margin: 0;
-    padding: 0;
+  .social-links a {
+    color: #dbd8d7;
+    width: 35px;
+    height: 35px;
+    padding: 2px 3px;
+    border-radius: 8px;
   }
 
-  nav li {
+  .social-links a:hover {
+    color: var(--default-font-color);
+  }
+
+  .grid-container {
+    display: grid;
+    padding: 120px 0;
+    align-items: center;
+    gap: 80px;
+  }
+
+  header {
+    color: var(--default-background-color);
+    font-family: var(--primary-font-family);
+    font-size: 40px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 52px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
+  }
+
+  .feature {
+    display: flex;
+    padding: 0 24px;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
+  }
```

Comment:
Kindly remove `padding: 0 24px;` from the feature div to fix the white space problem as shown in screenshot.

![Uploading Screenshot (178).png…]()